src/commands/sync/snapshot/README.md

```diff
@@ -24,6 +24,9 @@ The snapshot is cabable of containing following entities:
 - **Languages**
 - **Workflows**
 
+> [!IMPORTANT]
+> To successfully snapshot your environment, ensure your MAPI key has the necessary permissions. For more information about API keys, visit our [learn portal](https://kontent.ai/learn/docs/apis/openapi/management-api-v2/#tag/API-keys).
```

Comment:
It might be worth mentioning that for working with WS the "manage environments" permission is needed explicitly as it is quite confusing and probably the only confusing permission so it won't take mu cv space.